api/peering.go

```diff
@@ -211,3 +221,31 @@ func (p *Peerings) Initiate(ctx context.Context, i PeeringInitiateRequest, wq *W
 
 	return &out, wm, nil
 }
+
+func (p *Peerings) List(ctx context.Context, plr PeeringListRequest, q *QueryOptions) ([]*Peering, *QueryMeta, error) {
```

Comment:
I don't think we need `PeeringListRequest` here at all; the handler only expects EnterpriseMeta from the request which is set by `*QueryOptions`

Reply:
Maybe we could leave the request as-is in case we ever need to add to it in the future but get rid of the fields so it's clear they're not used